diff --git a/ableplayer.module b/ableplayer.module
--- a/ableplayer.module
+++ b/ableplayer.module
@@ -241,19 +241,19 @@ function ableplayer_modules_installed($modules) {
     }
   }
 
-  $factory = \Drupal::configFactory();
+  /* $factory = \Drupal::configFactory();
 
   $video_entity_form_display = $factory->getEditable('core.entity_form_display.media.video.default');
   $video_entity_form_display->set('content.field_ableplayer_media_caption.type', 'media_library_widget');
   $video_entity_form_display->save();
 
   $video_entity_view_display = $factory->getEditable('core.entity_view_display.media.video.default');
   $video_entity_view_display->set('content.field_media_video_file.type', 'ableplayer_video');
   $video_entity_view_display->set('content.field_media_video_file.settings', [
     'controls' => FALSE,
     'autoplay' => FALSE,
     'loop' => FALSE,
     'multiple_file_display_type' => 'tags',
   ]);
-  $video_entity_view_display->save();
+  $video_entity_view_display->save(); */
 }
